packages/@glimmer/syntax/test/loc-node-test.ts

```diff
@@ -314,6 +314,30 @@ data-barf="herpy"
   }
 });
 
+test('element block params', () => {
+  let ast = parse(`<Foo as |ab cd efg|></Foo>`);
+
+  let [Foo] = ast.body;
+  if (assertNodeType(Foo, 'ElementNode')) {
+    let [ab, cd, efg] = guardArray({ blockParamNodes: Foo.blockParamNodes }, { min: 3 });
+    locEqual(ab, 1, 8, 1, 12);
```

Comment:
since the signature for `locEqual` is
```js
function locEqual(
  node: AST.Node | null | undefined,
  startLine: number,
  startColumn: number,
  endLine: number,
  endColumn: number,
  message = JSON.stringify(node)
) {
```
should `endColumn` be 2 characters more than `startColumn`? right now it sees like.. 4?

Reply:
oh! thanks for noticing. since those are initially parsed as attributes by hbs parser the location contains the pipe as well... need to fix it. not sure why it also contains the whitespace, might be a bug in hbs parser
```js
't=<Foo as |ab cd efg|></Foo>'
t.slice(8,12)
'|ab '
t.slice(12,15)
'cd '
t.slice(15,19)
'efg|'
```